tests/Factory/AuthorizationServiceFactoryTest.php

```diff
@@ -31,7 +31,7 @@ public function testFactory()
     {
         $serviceManager = new ServiceManager();
 
-        $serviceManager->setService('Rbac\Rbac', $this->getMockBuilder('Rbac\Rbac')->disableOriginalConstructor()->getMock());
+        $serviceManager->setService('Rbac\Rbac', $this->getMockBuilder(\LmcRbacMvc\Rbac\Rbac::class)->disableOriginalConstructor()->getMock());
```

Comment:
@visto9259, the service call here is accessing LmcRbac\Rbac as defined [here.](https://github.com/ALTAMASH80/LmcRbacMvc/blob/remove-dependency-on-zfr-rbac/config/module.config.php#L26).

Reply:
I know but this is misleading.  Since we are removing references to `Rbac\Rbac`, we should not have a service that creates an object called `Rbac\Rbac`.

Reply:
If you want the Rbac\Rbac to be removed it may lead to a backward compatibility break. I kept it because I saw something similar in Laminas MVC to keep backward compatibility. If the service name should also be changed, I'll change it.